tests/manage/z_cluster/test_must_gather.py

```diff
@@ -51,3 +72,35 @@ def test_must_gather(self, mustgather, log_type):
         """
         mustgather.log_type = log_type
         mustgather.validate_must_gather()
+
+    @tier2
+    @bugzilla("1884546")
+    def test_must_gather_restart(self, mustgather_restart, nodes):
```

Comment:
Here we are restarting the node where must-gather is running, should we change the name according to it?
may be something like test_must_gather_pod_node_restart